crypto/cipher/aes_icm.c

```diff
@@ -98,6 +98,7 @@ static srtp_err_status_t srtp_aes_icm_alloc(srtp_cipher_t **c,
                                             int tlen)
 {
     srtp_aes_icm_ctx_t *icm;
+    (void)tlen;
```

Comment:
If this is just to suppress warnings about unused variables, we could just not give the parameter a name, I think.

Reply:
that gives a different warning ;)